src/scripts/worldmap/MapHelper.ts

```diff
@@ -285,6 +285,7 @@ class MapHelper {
             // Gain queue slots based on highest region
             App.game.breeding.gainQueueSlot(App.game.breeding.queueSlotsGainedFromRegion(player.highestRegion()));
             GameHelper.incrementObservable(player.highestRegion);
+            player.highestSubRegion = 0;
```

Comment:
I think this needs to make use of the following, as we don't have a setter/getter bound for it?
```suggestion
            player.highestSubRegion(0);
```

Reply:
> I think this needs to make use of the following, as we don't have a setter/getter bound for it?

fixed